test/guess-number.test.js

```diff
@@ -0,0 +1,43 @@
+import compareNumbers from '../src/guess-number.js';
+
+const test = QUnit.test;
+
+test('compare-number', (assert) => {
```

Comment:
I recommend using a more specific descriptor for your tests than just the same thing for all three - something like "returns -1 for a guess that's too low" or something like that would help you know what you're testing for without having to fully read the test.